File: BlackJack.WEB/ClientApp/src/app/shared/entities/game.views/get-details.game.view.ts
import { StartGameView } from "./start.game.view";
import { GetAllStepsGameView } from "./get-all-steps.game.view";
import { GetAllStepOfBotsGameView } from "./get-all-step-of-bots.game.view";

export class GetDetailsGameView {
  game?: StartGameView;
  playerSteps?: GetAllStepsGameView;
  botsSteps?: GetAllStepOfBotsGameView;

  constructor() {
    this.game = new StartGameView();
    this.playerSteps = new GetAllStepsGameView();
    this.botsSteps = new GetAllStepOfBotsGameView();
  }
}

File: BlackJack.ViewModels/GameViews/GetDetailsGameView.cs
namespace BlackJack.ViewModels.GameViews
{
    public class GetDetailsGameView
    {
        public StartGameView Game { get; set; }
        public GetAllStepsGameView PlayerSteps { get; set; }
        public GetAllStepOfBotsGameView BotsSteps { get; set; }
    }
}
